docs/packages/angular.md

```diff
@@ -0,0 +1,168 @@
+# @reatom/angular
+
+Angular bindings package for [Reatom](https://github.com/artalar/reatom) store.
+
+[![npm](https://img.shields.io/npm/v/@reatom/angular?style=flat-square)](https://www.npmjs.com/package/@reatom/angular)
+![npm type definitions](https://img.shields.io/npm/types/@reatom/angular?style=flat-square)
+[![npm bundle size](https://img.shields.io/bundlephobia/minzip/@reatom/angular?style=flat-square)](https://bundlephobia.com/result?p=@reatom/angular)
+![GitHub](https://img.shields.io/github/license/artalar/reatom?style=flat-square)
+
+[Open in docs](https://reatom.js.org/#/packages/angular)
+
+> Reatom is **declarative** and **reactive** state manager, designed for both simple and complex applications. See [docs](https://reatom.js.org/).
```

Comment:
```suggestion

```

Reply:
Thanks for the requested changes 👍
Now it seems to be much more clear and definitive